modules/lang-painless/src/main/java/org/elasticsearch/painless/node/PSubDefArray.java

```diff
@@ -86,6 +83,9 @@ void updateActual(Type actual) {
     @Override
     void setup(MethodWriter writer, Globals globals) {
         index.write(writer, globals);
+        writeIndexFlip(writer, w -> {
```

Comment:
I think you know I'm not a huge fan of lambdas :), but I fully understand why you do it here, since this is probably cleaner than another interface.